src/pages/p2p.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { WithIntl } from 'components/localization'
+import DP2PPage from 'features/pages/p2p'
+
+const DP2P = () => {
+    return <DP2PPage />
+}
+
+export default WithIntl()(DP2P)
```

Comment:
Please move it to index.tsx of p2p folder and move showing logic here

```suggestion
import React from 'react'
import { WithIntl } from 'components/localization'
import DP2PPage from 'features/pages/p2p'

const DP2P = () => {
    useOpenLiveChat(true)
    const { is_p2p_allowed_country } = useRegion()

    if (is_p2p_allowed_country) {
        return (
            <DP2PPage />
        )
    }
    return <PageNotFound />
}

export default WithIntl()(DP2P)
```

BTW I would to ask you @mohsen-deriv 🙏 
I noticed that @hasan-deriv used `TSmartContent` for this page while we never show this page for EU users. Isn't over engineering?